packages/functions/test/browser/callable.test.ts

```diff
@@ -18,36 +18,16 @@ import { expect } from 'chai';
 import * as sinon from 'sinon';
 import { FirebaseApp } from '@firebase/app-types';
 import { _FirebaseApp } from '@firebase/app-types/private';
-import { HttpsError, FunctionsErrorCode } from '@firebase/functions-types';
 import firebase from '@firebase/app';
-import '@firebase/messaging';
 import { Service } from '../../src/api/service';
+
+/* eslint-disable import/no-duplicates */
+import '@firebase/messaging';
```

Comment:
Just curious why the duplicate imports are needed.

Reply:
Technically, it's not needed, but I worry that hiding the bare import with the named import makes the code less clear and more error prone. E.g. people removes the named import without knowing that we rely on the side effect of the import.